OCP-4.X/cluster.yml

```diff
@@ -0,0 +1,26 @@
+---
+- hosts: orchestration
+  user: "{{ ORCHESTRATION_USER | default('root') }}"
+  roles:
+    - { role: ansible-config, when: OPENSHIFT_INSTALL | default(True) | bool }
+    - { role: cleanup, when: (OPENSHIFT_AWS_INSTALL_CLEANUP | default(False) | bool) or (OPENSHIFT_LIBVIRT_INSTALL_CLEANUP | default(False) | bool) }
+    - { role: install, when: OPENSHIFT_INSTALL | default(True) | bool }
+    - { role: quickstart, when: OPENSHIFT_INSTALL | default(True) | bool }
+    - { role: post-install, when: OPENSHIFT_POST_INSTALL | default(False) | bool }
+    - { role: post-config, when: OPENSHIFT_TOOLING | default(False) | bool }
```

Comment:
This is a nitpick but I am of the opinion that json nested in yaml is a bit more difficult to understand. I think simply converting the json into yaml provides a more readable playbook.

Ex:
```
- hosts: orchestration
  user: "{{ ORCHESTRATION_USER | default('root') }}"
  roles:
    - role: ansible-config
      when: OPENSHIFT_INSTALL | default(True) | bool
    - role: cleanup
      when: (OPENSHIFT_AWS_INSTALL_CLEANUP | default(False) | bool) or (OPENSHIFT_LIBVIRT_INSTALL_CLEANUP | default(False) | bool)
    - role: install
      when: OPENSHIFT_INSTALL | default(True) | bool
    - role: quickstart
      when: OPENSHIFT_INSTALL | default(True) | bool
    - role: post-install
      when: OPENSHIFT_POST_INSTALL | default(False) | bool
    - role: post-config
      when: OPENSHIFT_TOOLING | default(False) | bool
```
Let me know what you think.